fastlane_core/lib/fastlane_core/ui/interface.rb

```diff
@@ -118,9 +118,11 @@ class FastlaneCrash < StandardError
     # raised from user_error!
     class FastlaneError < StandardError
       attr_reader :show_github_issues
+      attr_reader :error_info
```

Comment:
Just a thought, but as we will receive an 'exception', could the instance variable could just be `:info`?